pyccel/ast/builtin_methods/list_methods.py

```diff
@@ -15,8 +15,9 @@
 
 
 __all__ = ('ListAppend',
+           'ListInsert',
+           'ListPop',
            'ListClear',
-           'ListPop'
```

Comment:
Please fix the pylint warning

Reply:
fixed in ecb58740e3f5b11a07555ef325f3933982e58160

Reply:
Thanks. FYI the order in `__all__` doesn't need to match the order of the classes in the file

Reply:
Noted, thanks.